.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Notion ticket link
+
+<!-- Please replace with your ticket's URL -->
+
+[Ticket Name](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7)
```

Comment:
What is the purpose of this?

Reply:
This is the template that other projects on Blueprint are using. The ticket name is a link back to the task on the Notion dev board.